lib/NativeCode/DynamicLibraryLoaderHelper/NativeRender/dllmain.cpp

```diff
@@ -1252,6 +1241,49 @@ static void eos_call_steam_init(const std::string& steam_dll_path)
     }
 }
 
+//-------------------------------------------------------------------------
+void eos_set_loglevel_via_config()
+{
+    if (EOS_Logging_SetLogLevel_ptr == nullptr)
+    {
+        return;
+    }
+
+    auto path_to_log_config_json = get_path_for_eos_service_config(EOS_LOGLEVEL_CONFIG_FILENAME);
```

Comment:
For now this is fine to have the code in one function, but preferably the function to 'read' the values from disk would be in it's own function, not in the same function with the code that sets the log level.